py/runtime.h

```diff
@@ -109,6 +110,46 @@ bool mp_sched_schedule(mp_obj_t function, mp_obj_t arg);
 bool mp_sched_schedule_node(mp_sched_node_t *node, mp_sched_callback_t callback);
 #endif
 
+// Handles any pending MicroPython events without waiting for an interrupt or
+// event.
+STATIC MP_ALWAYSINLINE inline void mp_event_handle_nowait(void) {
```

Comment:
When using static-inline the static must always be there, so STATIC should be static.

Write this either as:
```
MP_ALWAYSINLINE static inline ...
```
or:
```
static inline MP_ALWAYSINLINE ...
```

Reply:
Oh, I wondered why MicroPython had this `STATIC` macro. Is the idea that sometimes you want to build with more visible symbols?

Reply:
There's even a comment that explains it! I should have looked.

Reply:
Resolved (in py/event.h now)

Reply:
> Is the idea that sometimes you want to build with more visible symbols?

Yes, exactly.  Although I suspect there are better ways to do that with compiler/linker flags.  So I'd prefer to eventually move away from using our `STATIC` macro.